ui/app/src/modules/Preview/PreviewConcierge.tsx

```diff
@@ -388,10 +386,6 @@ export function PreviewConcierge(props: any) {
         );
       }
     }
-    const storedPage = getStoredPreviewToolsPanelPage(site, user.username);
-    if (storedPage) {
-      dispatch(pushToolsPanelPage(storedPage));
-    }
```

Comment:
Please test that the preferences are still stored/retrieved/applied as before.

Reply:
I did test and the ToolsPanelPage still works properly, it keeps the preferences (tool open, trees' states, etc)